corehq/apps/domain/templates/domain/admin/dhis2/dataset_map.html

```diff
@@ -4,16 +4,48 @@
 {% load crispy_forms_tags %}
 {% load i18n %}
 
+{% block js-inline %}{{ block.super }}
+    {% include 'domain/admin/partials/dhis2_dataset_map_templates.html' %}
+    <script src="{% static 'domain/js/dhis2_map_settings.js' %}"></script>
```

Comment:
Don't think these 2 should be in js-inline block. The templates should probably just live outside the block and the script should live in the `js` block.